net/golioth/fw.c

```diff
@@ -123,6 +123,10 @@ int golioth_fw_desired_parse(const uint8_t *payload, uint16_t payload_len,
 
 	err = zcbor_map_decode(zsd, map_entries, ARRAY_SIZE(map_entries));
 	if (err) {
+		if (payload_len == 1 && payload[0] == 0xa0) {
```

Comment:
Is there a macro for CoAP zero-length map? It would be nice to use that instead of the `0xa0` literal but I wasn't able to find it.